src/main/java/com/contrastsecurity/http/UrlBuilder.java

```diff
@@ -39,6 +39,10 @@ public String getApplicationsUrl(String organizationId) {
         return String.format("/ng/%s/applications?%s", organizationId, "base=false");
     }
 
+    public String getLicensedApplicationsUrl(String organizationId) {
+        return String.format("/ng/%s/applications%s", organizationId, "/filter?expand=license,compliance_policy,skip_links,technologies,trace_severity_breakdown,metadata&sort=appName&quickFilter=LICENSED");
```

Comment:
Can we cut out some(or most) of these expand parameters. Expands really hurt performance.

Reply:
Yep, took out all the expands, as the script does not seem to need that data. Will push changes shortly.